CHANGES.md

```diff
@@ -1,3 +1,62 @@
+# Version 0.5
+
+## New Features
+
+* Added experimental support for basic, non-compositional verification
+  of x86 machine code for use in conjunction with LLVM verification.
+
+        crucible_llvm_verify_x86 :
+          LLVMModule -> String -> String ->
+          [(String, Int)] -> Bool -> CrucibleSetup () ->
+          TopLevel CrucibleMethodSpec
+
+
+  The first argument specifies the LLVM module containing the _caller_.
+  The second and third specify the ELF file name and symbol name of the
+  function to be verifier. The fourth specifies the names and sizes (in
+  bytes) of global variables to initialize, and the fifth whether to
+  perform path satisfiability checking. The last argument is the LLVM
+  specification of the calling context against which to verify the
+  function.
+
+* Added support for using the SMT theory of arrays in the LLVM memory
+  model. In some cases, this can significantly improve performance.
+
+        enable_smt_array_memory_model : TopLevel ()
+        disable_smt_array_memory_model : TopLevel ()
+
+* Added experimental Verilog export, targeting the subset of Verilog
+  supported by ABC, in the new `write_verilog` and `offline_verilog`
+  commands.
+
+* Added several new options:
```

Comment:
New commands `enable_crucible_assert_then_assume` and `enable_crucible_assert_then_assume` that control whether a predicate is assumed after asserting it during the Crucible symbolic simulation. The default is `false`. This is a breaking change, previously this was set to `true`.